tests/torch/pruning/filter_pruning/test_algo.py

```diff
@@ -711,8 +711,8 @@ def test_flops_calculator(model_module, all_weights, pruning_flops_target, ref_f
     config["compression"]["algorithm"] = "filter_pruning"
     config["compression"]["params"]["all_weights"] = all_weights
     config["compression"]["params"]["prune_first_conv"] = True
-    config["compression"]["pruning_init"] = 0.5 if not pruning_flops_target else pruning_flops_target
-    if pruning_flops_target:
+    config["compression"]["pruning_init"] = pruning_flops_target if pruning_flops_target is not None else 0.5
+    if pruning_flops_target is not None:
```

Comment:
1. Before `pruning_init` would become 0.5 if `pruning_flops_target` **was not None**. Now it'll be equal 0.5 if `pruning_flops_target` **is None**. Was there a bug before?
2. Was this check designed for `pruning_flops_target` being None, 0 or both? Now it'll only be triggered by None.
3. What's now the correct way to check for 0 and None? Are checks of the form `if a:` are not accepted now?

Reply:
Yes, looks like it was a bug, because `bool(None)` and `bool(0.0)` == False,
But this changes does not effect on test (model PruningTestBatchedLinear final final pruning level is 0 in both case).